openupgrade_scripts/scripts/website/14.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def website_cookie_notice_pre_migration(env):
+    openupgrade.logged_query(
```

Comment:
Why making this? Please always add comments for explaining the why.

Reply:
Just renaming xmlids.

Reply:
Why do you need to make these renamings?

Reply:
Well, they have the same functionality. We always do that, right?

Reply:
Although the xmlids are noupdate=0, so In fact it doesn't matter, but is not wrong.

Reply:
Yeah, but in this case, renaming is just not so obvious. Starting with, due to the website COW process (create on write), there can be multiple copies of the view with the same key that are really used on the website, so this doesn't make the complete change (you would probably need to remove that copies).